data/application.css

```diff
@@ -74,6 +74,10 @@ toolbox revealer {
     padding: 6px;
 }
 
+.reset:hover {
+   background-color: rgba(29, 112, 213, 0.795);
```

Comment:
According to my calcs this should be `rgba(26, 100, 191, 0.795)`. But it depends on how do you make the original color lighter. So I'll merge it as is.